app/src/main/java/in/paperwrk/safetycollabproject/SplashActivity.java

```diff
@@ -4,25 +4,39 @@
 import android.os.Bundle;
 import android.os.Handler;
 import android.support.v7.app.AppCompatActivity;
+import android.widget.ProgressBar;
 
 public class SplashActivity extends AppCompatActivity {
 
+    private ProgressBar progressBar;
```

Comment:
Kindly follow class member naming convention and change 
`private ProgressBar progressBar` to `private ProgressBar mProgressBar`

Reply:
Yeah, actually the no code was present before. I will correct it. Thanks